modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/sql/fun/IgniteStdSqlOperatorTable.java

```diff
@@ -291,6 +292,9 @@ public IgniteStdSqlOperatorTable() {
         register(SqlStdOperatorTable.IS_NOT_JSON_ARRAY);
         register(SqlStdOperatorTable.IS_NOT_JSON_SCALAR);
 
+        // Aggregate functions.
+        register(SqlInternalOperators.LITERAL_AGG); // Internal operator, not implemented, required for serialization.
```

Comment:
What does it mean 'Not implemented' ?

Reply:
It's not implemented in RexImpTable, it processed in special way, by aggregates.